src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj

```diff
@@ -6,12 +6,12 @@
     <PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DevSpaces</PackageId>
 		<Description>Provides management operations support for Azure Dev Spaces.</Description>
-		<Version>0.11.0-preview</Version>
+		<Version>1.0.0</Version>
```

Comment:
@sanar-microsoft Are you sure you want to release a stable version at this point, we usually recommend releasing a couple of preview versions of the SDK for a particular version of the service so there is leeway to apply fixes without breaking end users/

Reply:
I'll change this to 0.12.0-preview then, test these bits and in case there are no more changes update the version to 1.0.0 in my next PR